proftpd/files/sftp.conf

```diff
@@ -0,0 +1,8 @@
+## File managed by Salt ##
+<IfModule mod_sftp.c>
+<VirtualHost 0.0.0.0>
+{% for param, value in sftp.items() -%}
+  {{param}} 	{{value }}
```

Comment:
here is where you have two spaces, which is why you need to verify those in your inspec test

Reply:
it was a space and a tabulation, I delete tab here and in inspec test too